src/Angor/Client/Models/ProjectValidationService.cs

```diff
@@ -0,0 +1,15 @@
+namespace Angor.Client.Models;
+
+public class ProjectValidationService
+{
+    public bool CanInvest(Project project)
```

Comment:
instead of a service what we did up to now is create a method on the Project class with the same logic